samples/bq_utils/README.md

```diff
@@ -0,0 +1,40 @@
+# Helper scripts for BigQuery validations
+
+## Dataset-level
+
+Currently, we do not natively support validations of an entire BQ dataset yet. This is a workaround to execute this task.
```

Comment:
I would remove this line because it's not on the roadmap. Usually customers want to do table-specific validations instead of only COUNT(*) validations, so we'll probably use this example unless there's more demand in the future

Reply:
I rewrote the sentence to just state that is not supported, please check what you think!